webroot/index.php

```diff
@@ -24,14 +24,14 @@
         return false;
     }
 }
-require dirname(__DIR__) . '/config/bootstrap.php';
+require dirname(__DIR__) . '/vendor/autoload.php';
```

Comment:
I tried to get 3.x installed through composer create-project (how?, could it be part of the regular readme?) to find my way around how/where Application.php is being executed because as @ravage84 points out it is not clear how/when/where boostrap.php is being loaded now.